pandas/src/parser/tokenizer.c

```diff
@@ -1675,23 +1483,20 @@ double xstrtod(const char *str, char **endptr, char decimal,
   num_digits = 0;
   num_decimals = 0;
 
-  // Process string of digits
-  while (isdigit(*p))
-  {
+  // Process string of digits.
```

Comment:
so this style seems to be prevalent. we don't have c-style checkers, so if you are changing this, nothing to stop someone else to change it back / another way.

can we clarify what is the correct style, maybe even add a checker in? (IIRC we should be using google c styles)